tests/raster/fixtures.py

```diff
@@ -1,28 +1,19 @@
-import os
 import pytest
 
 import click
 
-from kart import is_darwin
-
 # using a fixture instead of a skipif decorator means we get one aggregated skip
 # message rather than one per test
 @pytest.fixture(scope="session")
-def requires_gdal_info(data_archive_readonly):
-    from osgeo import gdal
-
-    # TODO - figure out what's wrong with gdal.Info in our macos build.
-    # We can't rely on the try/except block below since it completely crashes python.
-    if is_darwin and "CI" in os.environ:
-        raise pytest.skip("gdal.Info is not working on macOS on CI")
+def requires_gdal_info():
+    try:
+        from osgeo import gdal
 
-    with data_archive_readonly("raster/cog-aerial.tgz") as aerial:
-        try:
-            gdal.Info(f"{aerial}/aerial.tif")
-            has_gdal_info = True
-        except Exception as e:
-            click.echo(e, err=True)
-            has_gdal_info = False
+        assert gdal.Info is not None
+        has_gdal_info = True
+    except Exception as e:
```

Comment:
When would we have `gdal` but not `gdal.Info()`? I know we were trying to track down a bug, but do we need to keep this?

Reply:
Done